### ISSUE TITLE
CSS injection support

### ISSUE THREAD

**Issue by yuvadm** [thgh/rollup-plugin-css-only#20]
It seems the plugin only supports bundling css into a single `bundle.css` file.

Would it be possible to support injection of the styles into an existing `bundle.js`?

**Comment by thgh** [comment]
Yes, it's possible, I have seen it in another plugin. Happy to receive a PR!

**Comment by benmccann** [comment]
I'm doing something like that in https://github.com/domingues/rollup-plugin-css-chunks/pull/8

**Comment by socketopp** [comment]
I guess there is no library that does this today huh? Either I make a PR or just inject bundle.css manually.

**Comment by SonGokussj4** [comment]
I just found out about rollup and this plugin and i needed the injection because it'm writing tampermonkey script. It has to be all in 1 file. It there any update about this?
